packages/react-components/src/atoms/Tag.tsx

```diff
@@ -27,9 +27,9 @@ const styles = css({
   borderRadius: `${18 / perRem}em`,
 
   transition: '200ms',
-  ':hover': {
-    borderColor: charcoal.rgb,
-  },
+  // ':hover': {
```

Comment:
yaaaaay. I hated those, made it look like you're supposed to click ^^ why not remove it entirely though instead of commenting, incl. transition?

Reply:
I can remove it. I wanted to save time to my future self. This will come back.